xml/vk.xml

```diff
@@ -22165,6 +22216,27 @@ typedef void* <name>MTLSharedEvent_id</name>;
                 <enum value="&quot;VK_EXT_extension_528&quot;"          name="VK_EXT_EXTENSION_528_EXTENSION_NAME"/>
             </require>
         </extension>
+        <extension name="VK_QNX_external_memory_screen_buffer" number="529" type="device" author="QNX" depends="VK_KHR_sampler_ycbcr_conversion+VK_KHR_external_memory+VK_EXT_queue_family_foreign+VK_KHR_dedicated_allocation" platform="screen" contact="Mike Gorchak @mgorchak-blackberry, Aaron Ruby @aruby-blackberry" supported="disabled">
```

Comment:
Extension numbers [must be reserved in main branch](https://registry.khronos.org/vulkan/specs/1.3/styleguide.html#_registering_extensions_and_layers) before they can be used for an extension; in fact, this one is already reserved for a pending KHR extension.

Reply:
Well, I don't have an access to the internal gitlab repository. What is the first free number? I will submit a PR for the empty extension.

Reply:
You can use 530. I just synced up the internal repository XML with github.

Reply:
Thanks!